src/configuration/intro.md

```diff
@@ -1,74 +1,82 @@
-# Configuration
+## Introduction
 
-The core reads `~/.config/rizin/rizinrc` while starting. You can add `e` commands to this file to tune the rizin configuration to your taste.
+This chapter provides instructions on configuring and customizing Rizin to meet your preferences.
 
-To prevent rizin from parsing this file at startup, pass it the `-N` option.
+During the startup, Rizin loads a script file called `rizinrc`. Within this file, it is possible to incorporate configuration commands (`e` commands) to customize the tool.
```

Comment:
Please add a note, that the script file is just a sequence of the Rizin commands, so anything can be put here